BalloonBox-Inc/near-oracle-client
# [NEAR-2] NEAR wallet integration

I assert that the following checked items are true: :raised_hand:



- [x] The issue number(s) can be found in this PR's title 
  - [x] Acceptance criteria & description is up to date
  - [x] QA recommendation (regression, impacted modules) & instructions (including environment) are present
- [x] A PO (Matteo) has signed-off on this PR
- [x] This PR includes screenshots/screencasts

Ticket: [#2]

**Changes Proposed in this Pull Request:**

Connected to the NEAR testnet and initialized wallet based account. The wallet stores account keys in the browser's localStorage.

1) Connect button 
<img width="1436" alt="Screen Shot 2022-05-16 at 2 19 12 PM" src="https://user-images.githubusercontent.com/82831286/168684943-28ec11da-cd71-4f21-b26d-482e11c8d418.png">

2) Redirecting to the NEAR wallet SDK
<img width="1440" alt="Screen Shot 2022-05-16 at 2 19 22 PM" src="https://user-images.githubusercontent.com/82831286/168684357-be7042b8-37f5-4dc7-974c-8b749e3e592b.png">

3) Signed In (with the account id + disconnect button)
<img width="1436" alt="Screen Shot 2022-05-16 at 2 19 29 PM" src="https://user-images.githubusercontent.com/82831286/168684947-f8b04e6f-640c-4450-a3d3-1ca7235eb43a.png">

4) Storing account keys in the localstorage
<img width="497" alt="Screen Shot 2022-05-16 at 2 38 31 PM" src="https://user-images.githubusercontent.com/82831286/168686870-a23d5eda-730a-4f1d-a7b2-0f01b767f797.png">

<img width="1440" alt="Screen Shot 2022-05-16 at 2 19 43 PM" src="https://user-images.githubusercontent.com/82831286/168684386-e9dbad44-4293-4e18-a57f-620510aa2777.png">

**QA Recommendations**

1) Check if you are redirected to the NEAR wallet page and able to connect to your wallet once you click the connect button. 
2) Check if you can expand the button to see your account id and disconnect button once signed in.
3) Check if you can disconnect your wallet and see the success notification.
